generator/cluster_finder.hpp

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+#include "geometry/mercator.hpp"
+#include "geometry/rect2d.hpp"
+#include "geometry/tree4d.hpp"
+
+#include <functional>
+#include <queue>
+#include <set>
+#include <utility>
+#include <vector>
+
+namespace generator
+{
+// The class ClustersFinder finds clusters of objects for which IsSameFunc will return the true.
```

Comment:
s/the true/true/

optionally, s/will return/returns/

Reply:
++
++